GoF/game.py

```diff
@@ -3,30 +3,84 @@
 import numpy as np 
 import matplotlib.pyplot as plt 
 import matplotlib.animation as animation 
+import os
 
-# setting up the values for the grid 
+filename='test.txt'
+
+# setting up the values for the grid
 ON = 255
 OFF = 0
 vals = [ON, OFF] 
 
-def randomGrid(N): 
+def fill_from_file(i, j, grid, filename):
+	with open(os.path.join(os.path.dirname(__file__), 'resources', filename)) as f:
+		content = f.readlines()
+	# you may also want to remove whitespace characters like `\n` at the end of each line
+	content = [x.strip() for x in content]
+	for index in range(content.__len__()):
+		sep = content[index].index(',')
+		values = [content[index][0:sep], content[index][sep+1:content[index].__len__()]]
+		grid[int(values[0])+i, int(values[1])+i] = 255
+
+def read_pattern(pattern):
+	with open(os.path.join(os.path.dirname(__file__), 'resources', 'patterns', f'{pattern}.txt')) as f:

```

Comment:
please use np.load instead of manually reading. Save the file as the numpy array and then read it back